### PR TITLE
i updated my script to move to stage 2

### PR DESCRIPTION
**Pull request by bdairo**

### CODE REVIEW

File: scripts/bdairo.js [hunk 1]
```diff
@@ -1,3 +1,10 @@
 let name= "Boluwarin Dairo";
-let id = "INTERNSHIP ID: HNG-01402"
-console.log(" Hello World, This is " + name + " with " + id +" using javascript for stage 2 task" );
\ No newline at end of file
+let id = "INTERNSHIP ID: HNG-01402";
```
Comment (hng-teamfury) [inline scripts/bdairo.js]: Please remove the unnecessary text in your id variable.

```suggestion
let id = "HNG-01402";
```

File: scripts/bdairo.js [hunk 2]
```diff
@@ -1,3 +1,10 @@
 let name= "Boluwarin Dairo";
-let id = "INTERNSHIP ID: HNG-01402"
-console.log(" Hello World, This is " + name + " with " + id +" using javascript for stage 2 task" );
```
Comment (hng-teamfury) [inline scripts/bdairo.js:3]: You are missing your email from your output string. Also, there are some grammatical errors in your output (e.g. there is an unnecessary space at the beginning of your output string).

See [the goal](https://github.com/hng-teamfury/team-fury-1/blob/dev/README.md#the-goal) section of the README file for an example of how the output string should be formatted.

**Review by hng-teamfury — CHANGES_REQUESTED** [review]

**Review by Francisop — COMMENTED** [review]
var name = " ";
var email = " ";
var ID = "";
var language = "JavaScript";

console.log( "Hello world, " + "this is " + name + " with HNGi7 ID " + ID + " and email " + email + " using " + language  + " for stage 2 task";);

this format is better

**Comment by Francisop** [comment]
your code is too complicated

File: scripts/bdairo.js [hunk 3]
```diff
@@ -1,8 +1,11 @@
 let name= "Boluwarin Dairo";
-let id = "INTERNSHIP ID: HNG-01402";
+let id = "HNG-01402";
 let language="Javascript";
-let email = 'abisoyeoladayo@gmail.com';
-let output= " Hello World, This is " + name + " with " + id +" using " +language +" for stage 2 task";
+let email = 'dairobolu@gmail.com';
+let output= "Hello World, this is " + name + " with " + id +" and " + email +" using " + language +" for stage 2 task";
```
Comment (hng-teamfury) [inline scripts/bdairo.js]: It is still failing. If you look at [the goal example](https://github.com/hng-teamfury/team-fury-1/blob/master/README.md#the-goal) carefully, you will notice where your output differs.

So please fix accordingly.

**Pull request merged**